sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4797,6 +4797,8 @@ enqueueMemImageCommandHelper(pi_command_type CommandType, pi_queue Queue,
 
       // TODO: Level Zero does not support row_pitch/slice_pitch for images yet.
       // Check that SYCL RT did not want pitch larger than default.
+    (void)RowPitch;
+    (void)SlicePitch;
```

Comment:
Since you do not use these variables, why not... removing them? :-)

Such as having in the function definition just:
```c+
                             size_t /* RowPitch */, size_t /* SlicePitch */,
```

Reply:
They are used below inside the ASSERT, inside the DEBUG flag.

Reply:
I see. So probably your change is adequate.
I guess it will be ugly to add the name in the function definition according to the value of `NDEBUG`... :-)